package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "formsy-material-ui": "^0.6.0",
     "formsy-react": "^0.19.2",
+    "lodash.filter": "^4.6.0",
```

Comment:
I've just got a ❤️for lodash I guess.

Removing it for now, but I might sneak it in at a later stage ;)

Reply:
Will be removed in a rebase*

Reply:
I think using `Array.filter` is much cleaner than lodash / underscore :) Also much nicer to use the platform instead of dependencies!

`promises.filter(promise => promise.active`
vs
`_.filter(promises, promise => promise.active)`